### PR TITLE
Add FlexiBlas/3.0.4 with GCC/10.3.0 to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Adding FlexiBlas/3.0.4 with GCC/10.3.0 to NESSI/2022.11 again to avoid all past failures and reviews

### CODE REVIEW

**Comment by TopRichard** [comment]
bot: build I:AWS A:x86_64/generic R:11
bot: build I:AWS A:x86_64/intel/cascadelake R:11
bot: build I:AWS A:x86_64/intel/skylake R:11
bot: build I:Fram-NESSI A:x86_64/intel/broadwell R:11
bot: build I:eX3 A:zen2 R:11
bot: build I:eX3 A:aarch64/generic R:11

**Comment by TopRichard** [comment]
bot: build inst:AWS arch:x86_64/generic repo:11
bot: build inst:AWS arch:x86_64/intel/cascadelake repo:11
bot: build inst:AWS arch:x86_64/intel/skylake repo:11
bot: build inst:Fram-NESSI arch:x86_64/intel/broadwell repo:11
bot: build inst:eX3 arch:zen2 repo:11
bot: build inst:eX3 arch:aarch64/generic repo:11
- received bot command `build inst:AWS arch:x86_64/generic repo:11` from `TopRichard` (expanded command format: `build instance:AWS architecture:x86_64/generic repository:11`)
- received bot command `build inst:AWS arch:x86_64/intel/cascadelake repo:11` from `TopRichard` (expanded command format: `build instance:AWS architecture:x86_64/intel/cascadelake repository:11`)
- received bot command `build inst:AWS arch:x86_64/intel/skylake repo:11` from `TopRichard` (expanded command format: `build instance:AWS architecture:x86_64/intel/skylake repository:11`)
- received bot command `build inst:Fram-NESSI arch:x86_64/intel/broadwell repo:11` from `TopRichard` (expanded command format: `build instance:Fram-NESSI architecture:x86_64/intel/broadwell repository:11`)
- received bot command `build inst:eX3 arch:zen2 repo:11` from `TopRichard` (expanded command format: `build instance:eX3 architecture:zen2 repository:11`)
- received bot command `build inst:eX3 arch:aarch64/generic repo:11` from `TopRichard` (expanded command format: `build instance:eX3 architecture:aarch64/generic repository:11`)
- handling `build` resulted in: created jobs: 4609
- handling `build` resulted in: created jobs: 4610
- handling `build` resulted in: created jobs: 4611
- handling `build` resulted in: no jobs were prepared
- handling `build` resulted in: no jobs were prepared
- handling `build` resulted in: no jobs were prepared

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -441,12 +441,15 @@ $EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
...
-#fi
-#
+echo ">> Installing FlexiBlas-3.0.4-foss-2021a..."
+ok_msg="FlexiBlas. Nice!"
+fail_msg="Installation of FlexiBlas failed, that's unexpected..."
+if [ "${EESSI_SOFTWARE_SUBDIR}" = "ppc64le/generic" ]; then
+    $EB FlexiBLAS-3.0.4-GCC-10.3.0.eb --robot --ignore-test-failure
+else
+    $EB FlexiBLAS-3.0.4-GCC-10.3.0.eb --robot
+    check_exit_code $? "${ok_msg}" "${fail_msg}"
```
Comment (trz42) [inline EESSI-pilot-install-software.sh]: Before merging this line should be moved outside the if/fi block or the result of building for ppc64le/generic would not be checked.

**Review by trz42 — CHANGES_REQUESTED** [review]
Looks good. Small change is needed before merging.

**Comment by trz42** [comment]
Checklist for building: 

- [x]  did the builds for all required architectures succeed (SUCCESS message + reasonably sized tarball)
  - don't know about the tarball size yet, due to changed format of PR comment
- [x]  is the PR up-to-date with the target branch nessi.no-2022.11-dev in the repository (if not what are the differences)
- [x]  are all requested changes sound (checking files changed on GitHub.com)
  - Mostly yes. Small change is requested before merging. See https://github.com/NorESSI/software-layer/pull/89#pullrequestreview-1395072934
- [x]  is/are the easyconfig/s being built included with the EB version used (if not why not)
- [x]  were any changes needed to get the build(s) succeed (common changes for all architectures, changes for a single architecture, changes because of build environment specifics, etc.)
  - no changes were needed
- line <code>Checklist for building: 
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>- [x]  did the builds for all required architectures succeed (SUCCESS message + reasonably sized tarball)
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>  - don't know about the tarball size yet, due to changed format of PR comment
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>- [x]  is the PR up-to-date with the target branch nessi.no-2022.11-dev in the repository (if not what are the differences)
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>- [x]  are all requested changes sound (checking files changed on GitHub.com)
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>  - Mostly yes. Small change is requested before merging. See https://github.com/NorESSI/software-layer/pull/89#pullrequestreview-1395072934
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>- [x]  is/are the easyconfig/s being built included with the EB version used (if not why not)
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>- [x]  were any changes needed to get the build(s) succeed (common changes for all architectures, changes for a single architecture, changes because of build environment specifics, etc.)
</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line
- line <code>  - no changes were needed</code> is not considered to contain a bot command
- bot commands begin with `bot: `, make sure
  there is no whitespace at the beginning of a line

**Comment by trz42** [comment]
Checklist for deployment/ingestion

- [x] at least one tarball for each architecture has been uploaded
  - [x] `x86_64-intel-broadwell`
  - [x] `x86_64-intel-skylake_avx512`
  - [x] `x86_64-intel-cascadelake`
  - [x] `x86_64-amd-zen2`
  - [x] `x86_64-generic`
  - [x] `aarch64-generic`
- [x] at least one tarball for each architecture has been staged
  - [x] `x86_64-intel-broadwell`
  - [x] `x86_64-intel-skylake_avx512`
  - [x] `x86_64-intel-cascadelake`
  - [x] `x86_64-amd-zen2`
  - [x] `x86_64-generic`
  - [x] `aarch64-generic`
- [x] for each architecture **one and only one** tarball has been approved
  - [x] `x86_64-intel-broadwell`
  - [x] `x86_64-intel-skylake_avx512`
  - [x] `x86_64-intel-cascadelake`
  - [x] `x86_64-amd-zen2`
  - [x] `x86_64-generic`
  - [x] `aarch64-generic`
- [x] for each architecture **one and only one** tarball was ingested
  - [x] `x86_64-intel-broadwell`
  - [x] `x86_64-intel-skylake_avx512` 
  - [x] `x86_64-intel-cascadelake`
  - [x] `x86_64-amd-zen2`
  - [x] `x86_64-generic`
  - [x] `aarch64-generic`
- [x] has the package become available via CernVM-FS for all architectures ingested (for lmod cache updates pay attention to timestamp of files)
  **command**
  ```
  ls -l /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/{generic,amd/zen2,intel/[bsc]*}/{software,modules/all}/FlexiBLAS/*
  ls -l /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/aarch64/generic/{software,modules/all}/FlexiBLAS/*
  ```
  **log**
  ```
  #ls -l /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/{generic,amd/zen2,intel/[bsc]*}/{software,modules/all}/FlexiBLAS/*
   -rw-r--r--. 1 cvmfs cvmfs 1534 Apr 20 10:22 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/amd/zen2/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua
  -rw-r--r--. 1 cvmfs cvmfs 1533 Apr 20 09:43 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/generic/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua
  -rw-rw-r--. 1 cvmfs cvmfs 1541 Apr 20 12:02 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/broadwell/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua
  -rw-r--r--. 1 cvmfs cvmfs 1543 Apr 20 09:35 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/cascadelake/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua
  -rw-r--r--. 1 cvmfs cvmfs 1546 Apr 20 09:38 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/skylake_avx512/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/amd/zen2/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 10:17 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 10:22 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 10:17 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 10:17 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 10:22 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 10:22 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 10:17 share

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/generic/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 09:40 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:43 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:40 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:40 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 09:43 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 09:43 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:40 share

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/broadwell/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 11:56 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 12:02 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 11:56 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 11:56 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 12:02 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 12:02 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 11:56 share

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/cascadelake/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 09:34 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:35 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:34 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:34 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 09:35 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 09:35 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:34 share

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/x86_64/intel/skylake_avx512/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 09:35 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:38 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:35 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:35 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 09:37 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 09:37 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:35 share

  # ls -l /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/aarch64/generic/{software,modules/all}/FlexiBLAS/*
  -rw-r--r--. 1 cvmfs cvmfs 1534 Apr 20 10:05 /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/aarch64/generic/modules/all/FlexiBLAS/3.0.4-GCC-10.3.0.lua

  /cvmfs/pilot.nessi.no/versions/2022.11/software/linux/aarch64/generic/software/FlexiBLAS/3.0.4-GCC-10.3.0:
  total 28
  drwxr-xr-x. 2 cvmfs cvmfs 4096 Apr 20 09:56 bin
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 10:06 easybuild
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:56 etc
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:56 include
  drwxr-xr-x. 5 cvmfs cvmfs 4096 Apr 20 10:05 lib
  lrwxrwxrwx. 1 cvmfs cvmfs    3 Apr 20 10:05 lib64 -> lib
  drwxr-xr-x. 3 cvmfs cvmfs 4096 Apr 20 09:56 share
  ```

**Comment by trz42** [comment]
Test if bots are still noisy?

**Comment by trz42** [comment]
bot: help
- received bot command `help` from `trz42` (expanded command format: `help `)

**Review by trz42 — APPROVED** [review]

**Pull request merged**